sympy/integrals/laplace.py

```diff
@@ -1561,12 +1608,188 @@ def _inverse_laplace_time_diff(F, s, t, plane):
     return None
 
 
+def _inverse_laplace_irrational(fn, s, t, plane):
+    """
+    Helper function for the class InverseLaplaceTransform.
+    """
+
+    a = Wild('a', exclude=[s])
+    b = Wild('b', exclude=[s])
+    m = Wild('m', exclude=[s])
+    n = Wild('n', exclude=[s])
+
+    result = None
+    condition = S.true
+
+    fa = fn.as_ordered_factors()
+
+    ma = [x.match((a*s**m+b)**n) for x in fa]
+
+    if (None in ma) == 1:
```

Comment:
What does this mean?

Reply:
It means that I had a fall-back into my Perl heydays, testing `True` with `== 1`.  It should be `if None in ma:`, and it checks whether anything in `fa` fails to match `(a*s**m+b)**n`. Then we already know we will not have a rule and give up right away.

Reply:
Done.